### PR TITLE
Fixes #2528 - NTRs - externa and interna theca cells 

### LINKED ISSUE #2528: [NTR] Adding subclasses to theca cell
Please check that the term does not already exist by using the ontology search tool OLS:
https://www.ebi.ac.uk/ols4/ontologies/cl

*Preferred term label*
**interna theca cell**

**Synonyms** (add reference(s), please)
interna TC [PMID: 31320652](https://pubmed.ncbi.nlm.nih.gov/31320652/)
inTC [PMID: 31320652](https://pubmed.ncbi.nlm.nih.gov/31320652/)

**Definition** (free text, with reference(s), please. PubMed ID format is PMID:XXXXXX)
A specialized theca cell that forms the inner, highly vascularized layer of the theca surrounding ovarian follicles. Originating from progenitor theca cells, the theca interna cell is steroidogenic, playing a crucial role in the production of androgens, particularly androstenedione, which serves as a precursor for estrogen synthesis in granulosa cells. This cell expresses luteinizing hormone receptors, enabling it to respond to hormonal signals that regulate steroidogenesis.

[PMID: 31320652](https://pubmed.ncbi.nlm.nih.gov/31320652/)
PMID: [32530882](https://pubmed.ncbi.nlm.nih.gov/32530882)
https://doi.org/10.1016/j.biocel.2005.01.016
Wikipedia:Theca interna

**Parent cell type term** (check the hierarchy here https://www.ebi.ac.uk/ols4/ontologies/cl)
[theca cell](http://purl.obolibrary.org/obo/CL_0000503)

**Anatomical structure where the cell type is found** (check Uberon for anatomical structures: https://www.ebi.ac.uk/ols4/ontologies/uberon)
[theca interna](http://purl.obolibrary.org/obo/UBERON_0000157)

**Your ORCID**

**Additional notes or concerns**
logical def - EquivalenTo: 'theca cell' and 'part of' some 'theca interna'
Subclass of: 'androgen secreting cell' ?

---------

Please check that the term does not already exist by using the ontology search tool OLS:
https://www.ebi.ac.uk/ols4/ontologies/cl

*Preferred term label*
**externa theca cell**

**Synonyms** (add reference(s), please)
externa TC [PMID: 31320652](https://pubmed.ncbi.nlm.nih.gov/31320652/)
exTC [PMID: 31320652](https://pubmed.ncbi.nlm.nih.gov/31320652/)

**Definition** (free text, with reference(s), please. PubMed ID format is PMID:XXXXXX)
A specialized theca cell that forms the outer layer of the theca around the ovarian follicle, appearing at the antral follicle. Originating  from progenitor theca cells, theca externa cell is characterized by its fibroblast-like appearance and primarily function to provide structural support to the developing follicle. This cell produces collagen fibers and extracellular matrix components such as Col1a1 and Col1a2.
PMID: [32530882](https://pubmed.ncbi.nlm.nih.gov/32530882)
[PMID: 31320652](https://pubmed.ncbi.nlm.nih.gov/31320652/)
PMID: 36758341
Wikipedia:Theca externa

**Parent cell type term** (check the hierarchy here https://www.ebi.ac.uk/ols4/ontologies/cl)
[theca cell](http://purl.obolibrary.org/obo/CL_0000503)
connective tissue cell
theca cell 

**Anatomical structure where the cell type is found** (check Uberon for anatomical structures: https://www.ebi.ac.uk/ols4/ontologies/uberon)
[theca externa](http://purl.obolibrary.org/obo/UBERON_0000156)

**Your ORCID**

**Additional notes or concerns**

### PR DESCRIPTION
**Pull request by Caroline-99**
Fixes #2528

-Added two new terms - interna and exerna theca cells 
-Added textual, logical defs, references and synonyms -imported uberon terms
-revised the logical and textual defs of theca cell

### CODE REVIEW

**Comment by aleixpuigb** [comment]
#gogoeditdiff

**Comment by Caroline-99** [comment]
#gogoeditdiff

File: src/ontology/cl-edit.owl [hunk 1]
```diff
@@ -7737,11 +7741,11 @@ SubClassOf(obo:CL_0000502 ObjectSomeValuesFrom(obo:RO_0002215 obo:GO_0070253))
 
 # Class: obo:CL_0000503 (theca cell)
 
-AnnotationAssertion(Annotation(oboInOwl:hasDbXref "MESH:D013799") obo:IAO_0000115 obo:CL_0000503 "A flattened stroma cell forming a sheath or theca outside the basal lamina lining the mature ovarian follicle. Thecal interstitial or stromal cells are steroidogenic, and produce primarily androgens which serve as precusors of estrogens in the granulosa cells.")
+AnnotationAssertion(Annotation(oboInOwl:hasDbXref "MESH:D013799") Annotation(oboInOwl:hasDbXref "PMID:36758341") Annotation(oboInOwl:hasDbXref "doi:/10.1016/j.biocel.2005.01.016") obo:IAO_0000115 obo:CL_0000503 "A specialized stromal cell that forms the theca layer outside the basal lamina lining the ovarian follicle, appearing during the secondary follicle stage.")
```
Comment (aleixpuigb) [inline src/ontology/cl-edit.owl:7744]: For doi:/10.1016/j.biocel.2005.01.016 you can use the PMID:15833266

File: src/ontology/cl-edit.owl [hunk 2]
```diff
@@ -32727,6 +32731,31 @@ AnnotationAssertion(Annotation(oboInOwl:hasDbXref "PMID:7107724") oboInOwl:hasRe
 AnnotationAssertion(rdfs:label obo:CL_4052011 "follicle associated enterocyte")
 SubClassOf(obo:CL_4052011 obo:CL_0000584)
 
+# Class: obo:CL_4052012 (interna theca cell)
+
+AnnotationAssertion(Annotation(oboInOwl:hasDbXref "PMID:31320652") Annotation(oboInOwl:hasDbXref "PMID:32530882") Annotation(oboInOwl:hasDbXref "Wikipedia:Theca_interna") Annotation(oboInOwl:hasDbXref "doi:/10.1016/j.biocel.2005.01.016") obo:IAO_0000115 obo:CL_4052012 "A specialized theca cell that forms the inner, highly vascularized layer of the theca surrounding ovarian follicles. Originating from progenitor theca cells, the theca interna cell is steroidogenic, playing a crucial role in the production of androgens, which serves as a precursor for estrogen synthesis in granulosa cells. This cell expresses luteinizing hormone receptors, enabling it to respond to hormonal signals that regulate steroidogenesis.")
```
Comment (aleixpuigb) [inline src/ontology/cl-edit.owl:32736]: For doi:/10.1016/j.biocel.2005.01.016 you can use the PMID:15833266

File: src/ontology/cl-edit.owl [hunk 3]
```diff
@@ -32727,6 +32731,31 @@ AnnotationAssertion(Annotation(oboInOwl:hasDbXref "PMID:7107724") oboInOwl:hasRe
...
+AnnotationAssertion(terms:contributor obo:CL_4052013 <https://orcid.org/0009-0000-8480-9277>)
+AnnotationAssertion(terms:date obo:CL_4052013 "2024-09-30T11:10:15Z"^^xsd:dateTime)
+AnnotationAssertion(Annotation(oboInOwl:hasDbXref "PMID:31320652") Annotation(oboInOwl:hasSynonymType obo:OMO_0003000) oboInOwl:hasRelatedSynonym obo:CL_4052013 "exTC")
+AnnotationAssertion(Annotation(oboInOwl:hasDbXref "PMID:31320652") Annotation(oboInOwl:hasSynonymType obo:OMO_0003000) oboInOwl:hasRelatedSynonym obo:CL_4052013 "externa TC")
+AnnotationAssertion(rdfs:label obo:CL_4052013 "externa theca cell")
+EquivalentClasses(obo:CL_4052013 ObjectIntersectionOf(obo:CL_0000503 ObjectSomeValuesFrom(obo:BFO_0000050 obo:UBERON_0000156)))
+SubClassOf(obo:CL_4052013 obo:CL_0002320)
```
Comment (aleixpuigb) [inline src/ontology/cl-edit.owl:32756]: This is not needed since it is inherited (theca cell is a stromal cell, which is a connective tissue cell)

**Review by aleixpuigb — CHANGES_REQUESTED** [review]
Please, see inline comments

**Review by aleixpuigb — APPROVED** [review]

**Pull request merged**